modules/git/init.zsh

```diff
@@ -71,7 +71,7 @@ alias gf='git fetch'
 alias gfc='git clone'
 alias gfm='git pull'
 alias gfr='git pull --rebase'
-alias gfu='git remote update -p; git merge --ff-only @\{u\}'
+alias gfu='git fetch --all --prune; git merge --ff-only @\{u\}'
```

Comment:
Can we change the `;` to `&&` so the merge only gets called if the fetch succeeded?